models/articles.models.js

```diff
@@ -3,9 +3,14 @@ const {
   articlesSortByCheck,
   articlesOrderByCheck,
 } = require("../db/seeds/utils");
-exports.fetchArticles = (sort_by = "created_at", order_by = "DESC") => {
+exports.fetchArticles = (
+  sort_by = "created_at",
+  order_by = "DESC",
+  topic = "topic"
```

Comment:
No need for a default value for topic. If a topic is not passed in, it should just retrieve all the articles